Gemfile.lock

```diff
@@ -210,35 +209,47 @@ GEM
       rails (>= 3.0)
     globalid (0.3.6)
       activesupport (>= 4.1.0)
-    google-api-client (0.7.1)
-      addressable (>= 2.3.2)
-      autoparse (>= 0.3.3)
-      extlib (>= 0.9.15)
-      faraday (>= 0.9.0)
-      jwt (>= 0.1.5)
-      launchy (>= 2.1.1)
-      multi_json (>= 1.0.0)
-      retriable (>= 1.4)
-      signet (>= 0.5.0)
-      uuidtools (>= 2.1.0)
-    guard (2.6.1)
+    google-api-client (0.8.6)
```

Comment:
My rationale here is simple: the Gemfile specified ~> versions. If that's not acceptable, don't specify versions in an optimistic way :)

We don't use all these gems in our production site but we do use a lot of them, and I tested the PR on our environment before submitting. I can look at manually testing out agents that use any upgrades gems.

Truly, though, if it's a risk to let gems change, they should be pinned to exact version tags or commit refs. Someone somewhere has to have faith in upstream gems and tests to make up for misplaced faith. :grin: